src/rules/twoslash.test.ts

```diff
@@ -5,26 +5,6 @@ import { filename, ruleTester } from "./ruleTester.js";
 
 ruleTester.run("expect", expect, {
 	invalid: [
-		{
-			code: dedent`
-        const square = (x: number) => x * x;
-        const four = square(2);
-        //    ^? const four: string
-      `,
-			errors: [
-				{
-					column: 7,
-					line: 2,
-					messageId: "TypesDoNotMatch",
-				},
-			],
-			filename,
-			output: dedent`
-        const square = (x: number) => x * x;
-        const four = square(2);
-        //    ^? const four: number
-      `,
-		},
```

Comment:
The removed test is a duplicate of another one below: https://github.com/JoshuaKGoldberg/eslint-plugin-expect-type/blob/bf98512136f299226120c20abe578c5b337cef67/src/rules/twoslash.test.ts#L102-L121